apps/back/src/DataFixtures/CourseUserFixtures.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\CourseUser;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class CourseUserFixtures extends Fixture
+{
+    public function load(ObjectManager $manager): void
+    {
+        /* Get necessary data to create users (ex: is admin) */
+        $courseUserData = self::getCourseUserData();
+        $i = 1;
+
+        /* Foreach user, create user data and persist it */
+        foreach ($courseUserData as $userData) {
+            $user = new CourseUser();
```

Comment:
Where is the migration to create this table ?

Reply:
Ce serait pas plutôt que `CourseUser` est juste devenu `User` ?